card/spec/lib/card/content_spec.rb

```diff
@@ -191,11 +191,15 @@
       @card = card
 
       # non-nil valued opts only ...
-      @render_block = proc do |opts|
-        options = opts.inject({}) do |i, v|
-          i if !v[1].nil? && (i[v[0]] = v[1])
+      @render_block = proc do |chunk|
```

Comment:
this is harder to understand than the actual code.

Reply:
The only goal was to get the tests pass again. Make a story for it with at least 2 points then I'll make it beautiful :sunglasses: